src/main/webapp/script.js

```diff
@@ -90,23 +118,18 @@ function displaySentimentBucketChart(sentimentBucketList) {
     'bar': {groupWidth: '100'},
     'tooltip': {isHtml: true},
   };
-
   // Hide loading image once chart is drawn
   document.getElementById('loading-img').style.display = 'none';
 
   const view = new google.visualization.DataView(CommentSentimentTable);
   const chart = new google.visualization.ColumnChart(
       document.getElementById('chart-container'));
   chart.draw(view, options);
-
-  const averageContainer = document.getElementById('average-score-container');
-  averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore;
 }
 
 /**
  * Create a word cloud based on the number of appearance for each word
- * @param {Map<String:Integer>} wordFrequencyMap Map that contains
- *                                top popular words and its appearance
+ * @param {Map<String:Integer>} wordFrequencyMap that contains top words
```

Comment:
change it to Map instead of Map<String:Integer> to resolve this error

Reply:
ohhhh gotcha ty!!